examples/minimal-mdns/client.cpp

```diff
@@ -40,12 +39,13 @@ namespace {
 
 struct Options
 {
-    bool enableIpV4        = false;
-    bool unicastAnswers    = true;
-    uint32_t runtimeMs     = 500;
-    uint16_t querySendPort = 5353;
-    uint16_t listenPort    = 5388;
-    const char * query     = "_services._dns-sd._udp.local";
+    bool enableIpV4           = false;
+    bool unicastAnswers       = true;
+    uint32_t runtimeMs        = 500;
+    uint16_t querySendPort    = 5353;
+    uint16_t listenPort       = 5388;
```

Comment:
This is for the sample client. It should technically be random, however I was unsure how chip::Inet actually can 'bind to random port' so I made it as an argument.

Unsure what we do in the real app. I expect that we will want to be both client and server so we will be on 5353 all the time and never use the 'random' bit (since we can ask for unicast replies via flags)